core/src/test/scala/org/apache/spark/JobCancellationSuite.scala

```diff
@@ -320,6 +321,63 @@ class JobCancellationSuite extends SparkFunSuite with Matchers with BeforeAndAft
     f2.get()
   }
 
+  test("Interruptible iterator of shuffle reader") {
+    // In this test case, we create a Spark job of two stages. The second stage is cancelled during
+    // execution and a counter is used to make sure that the corresponding tasks are indeed
+    // cancelled.
+    import JobCancellationSuite._
+    val numSlice = 1
```

Comment:
can we hardcode it? using a variable makes people feel like they can change its value and the test can still pass, however it's not true as `assert(executionOfInterruptibleCounter.get() <= 10)` needs to be updated too.

Reply:
Will update it later. 

But looks like Jenkins are having troubles there days? it it back to normal?

Reply:
I'm not sure, let's just try it :)